.github/workflows/ci.yml

```diff
@@ -78,7 +79,7 @@ jobs:
     name: Windows Server 2019, Erlang/OTP ${{ matrix.otp_version }}
     strategy:
       matrix:
-        otp_version: ['24', '25']
+        otp_version: ['24', '25', '26']
```

Comment:
I wasn't sure about windows, should we just add the new one, or would it be enough to just keep 24 and 26?

Reply:
Getting a very weird error on [windows](https://github.com/sabiwara/elixir/actions/runs/4997496256/jobs/8952117592), should probably be investigated separately:

<img width="589" alt="Screenshot 2023-05-17 at 8 17 46" src="https://github.com/elixir-lang/elixir/assets/11598866/c7e8ef77-15f2-4143-a4b9-16a47d72290b">

If I understand correctly, it means that `Application.get_env(:elixir, :ansi_enabled, false)` [here](https://github.com/elixir-lang/elixir/blob/main/lib/elixir/lib/io/ansi.ex#L69) returns `:ebadf`, maybe a bug on OTP26 for windows? 🤔

Reply:
I'm trying to reproduce the above, seems to be related to a corner case in windows fileno API.

Reply:
tl;dr is: if application is started without a console, `_fileno` returns `-2` without erroring, which then fails when trying to check if its a tty. Since erlang is properly handling this error here: https://github.com/erlang/otp/blame/master/erts/emulator/nifs/common/prim_tty_nif.c#L211 I'm not sure if elixir should simply handle this (as I did here: https://github.com/v0idpwn/elixir/commit/da365c9dfa03c4996f3fdb4e1afdf9cd91b0fae3) or this should be considered a bug in erlang :)

Reply:
Please send a PR for Elixir and report a bug in Erlang. :)

Reply:
Can you please rebase @sabiwara?